src/storage.js

```diff
@@ -0,0 +1,122 @@
+//!This file contains functions related to checking for, adding and deleting `.comment` files and `.comments` directories.
+
+/*
+ * c
+ * https://github.com/rumpl/c
+ *
+ * Copyright (c) 2012 Djordje Lukic
+ * Licensed under the MIT license.
+ */
+
+"use strict";
+
+var fs = require("fs"); //FileSystem
+var path = require("path"); //Paths
+var underscore = require("underscore"); //Some functional shit
+var storage = module.exports;
+var DIRECTORY = ".comments";
+var EXTENSION = ".comment";
+
+/** Sets a `.comment` file for a specific file.
+ * @param {File} file a provided directory from the file tree.
+ * @param {String} comment The comment to be written.
+ */
+storage.set = function (file, comment) {
+  //Check if `.comments` exists, makes it if not.
+  if (!storage.exists(path.dirname(file))) {
+    storage.create(path.dirname(file));
+  }
+
+  var commentsFile = getCommentsFile(file); //Gets the file path
+  fs.writeFileSync(commentsFile, comment); //Writes the .comment file.
+  console.log(file + ".comment was written successfully.");
```

Comment:
The `storage` should be a low-level thing that doesn't interact with the user, it's the cli that should show the information/warnings/errors to the user. The storage should only manage storage.

Reply:
I fixed this in a few other places throughout the storage file as well.